packages/lib-react-components/src/ZooFooter/components/LinkList/TwoColumnLinkList.js

```diff
@@ -0,0 +1,53 @@
+import { Anchor, Box } from 'grommet'
+import { arrayOf, string } from 'prop-types'
+import styled from 'styled-components'
+
+import zipLabelsAndUrls from '../../helpers/zipLabelsAndUrls'
+import SpacedText from '../../../SpacedText'
+
+const StyledBox = styled(Box)`
+  height: fit-content;
+  list-style-type: none;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: repeat(6, auto);
+  column-gap: 8px;
+`
+
+export default function TwoColumnLinkList({ className, labels, urls }) {
```

Comment:
`className` isn't used, not sure if want to include in jsx, remove from props, or leave as is (not causing an issue).